[2023-01-13 09:52:47,439] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: my_first_dag_v3.third_task scheduled__2023-01-05T00:00:00+00:00 [queued]>
[2023-01-13 09:52:47,530] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: my_first_dag_v3.third_task scheduled__2023-01-05T00:00:00+00:00 [queued]>
[2023-01-13 09:52:47,530] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2023-01-13 09:52:47,530] {taskinstance.py:1242} INFO - Starting attempt 1 of 6
[2023-01-13 09:52:47,530] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2023-01-13 09:52:47,553] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): third_task> on 2023-01-05 00:00:00+00:00
[2023-01-13 09:52:47,567] {standard_task_runner.py:52} INFO - Started process 1756 to run task
[2023-01-13 09:52:47,589] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag_v3', 'third_task', 'scheduled__2023-01-05T00:00:00+00:00', '--job-id', '51', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dag.py', '--cfg-path', '/tmp/tmpxhv_bg2m', '--error-file', '/tmp/tmp34lwb5xp']
[2023-01-13 09:52:47,604] {standard_task_runner.py:77} INFO - Job 51: Subtask third_task
[2023-01-13 09:52:47,757] {logging_mixin.py:109} INFO - Running <TaskInstance: my_first_dag_v3.third_task scheduled__2023-01-05T00:00:00+00:00 [running]> on host ffa830cbd52d
[2023-01-13 09:52:47,908] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=salman
AIRFLOW_CTX_DAG_ID=my_first_dag_v3
AIRFLOW_CTX_TASK_ID=third_task
AIRFLOW_CTX_EXECUTION_DATE=2023-01-05T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-05T00:00:00+00:00
[2023-01-13 09:52:47,909] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-01-13 09:52:47,910] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am third task and will be ececuted after first task!']
[2023-01-13 09:52:47,948] {subprocess.py:85} INFO - Output:
[2023-01-13 09:52:47,964] {subprocess.py:89} INFO - hey, I am third task and will be ececuted after first task!
[2023-01-13 09:52:47,967] {subprocess.py:93} INFO - Command exited with return code 0
[2023-01-13 09:52:48,020] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=my_first_dag_v3, task_id=third_task, execution_date=20230105T000000, start_date=20230113T095247, end_date=20230113T095248
[2023-01-13 09:52:48,082] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-01-13 09:52:48,270] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
